oracle/provider/binance.go

```diff
@@ -239,7 +239,11 @@ func (p *BinanceProvider) getCandlePrices(key string) ([]types.CandlePrice, erro
 	return candleList, nil
 }
 
-func (p *BinanceProvider) messageReceived(_ int, bz []byte) {
+func (p *BinanceProvider) messageReceived(messageType int, bz []byte) {
+	if messageType != websocket.TextMessage {
```

Comment:
Is the expected message type for all these providers `websocket.TextMessage`? iirc a lot of these don't follow the expected set of rules

Reply:
This is how this check was set before we got rid of it. I'll test out all the providers though before merging to make sure it still works.

Reply:
👍 thank you, sounds good